src/Linters/ASTLinter.hack

```diff
@@ -9,14 +9,15 @@
 
 namespace Facebook\HHAST;
 
-use namespace Facebook\{HHAST, TypeAssert};
 use namespace Facebook\HHAST\SuppressASTLinter;
 use namespace HH\Lib\Vec;
 
 abstract class ASTLinter extends BaseLinter {
+  <<__Enforceable, __Reifiable>>
```

Comment:
This is a breaking change.
You could previously target `ListItem<SomeNode>` without a typechecker error.
However, this was a bug imo, since the inner type was effectively ignored.
`ListItem<Node>` is a valid type which is no longer supported.